src/khoj/utils/constants.py

```diff
@@ -49,9 +49,9 @@
         },
         "github": {
             "pat-token": None,
-            "repo-name": None,
-            "repo-owner": None,
-            "repo-branch": "master",
+            "repos": [
+                {"name": "khoj", "owner": "khoj-ai", "branch": "master"},
+            ],
```

Comment:
Shouldn't the repos list be empty, instead of containing the khoj repository by default?
```suggestion
            "repos": [],
```